python/tvm/ir/json_compact.py

```diff
@@ -57,6 +57,53 @@ def _updater(data):
     return _updater
 
 
+def create_updater_08_to_09():
+    """
+    Create an update to upgrade json from v0.8 to v0.9
+
+    Returns
+    -------
+    fupdater : function
+        The updater function
+    """
+
+    def _initialize_virtual_device(item, _):
+        if "virtual_device_" not in item["attrs"].keys():
```

Comment:
`"virtual_device_" not in item["attrs"]` is better so it do ont have to instantiate keys.

Reply:
Done! Thanks for the review